bin/generate-feature-config.php

```diff
@@ -35,6 +35,10 @@
 	}
 	$write .= "\t\t\t)";
 
+	if ( is_file( './dist' ) || ! wp_mkdir_p( './dist' ) ) {
```

Comment:
This file isn't run in a WP context, so this function doesn't exist. What about doing something like this:

```suggestion
	if ( ! is_dir( './dist' ) && ! @mkdir( './dist' ) ) {
```

It'll error if `dist` is a file for some reason, but that _is_ an error case so I think that's an OK option. Maybe we can add a message here to suggest running `npm run clean` to wipe artifacts?

Reply:
@ryelle I have updated the PR with your suggestions. If you 

`rm -rf dist && touch dist && npm run build`

you'll see that it errors out but the message I added is not in the output or in the log. Would it be better to start the build with `npm run clean`?

Reply:
I do see the error, but it's missing a linebreak:

<img width="551" alt="Screen Shot 2019-04-24 at 2 39 37 PM" src="https://user-images.githubusercontent.com/541093/56684980-d66f6000-669e-11e9-97ba-f752b03facb7.png">

Reply:
@ryelle Thanks, I have updated the PR & added two line breaks so there is a blank line after the message.